packages/web/src/scss/components/Toast/_ToastBar.scss

```diff
@@ -58,21 +58,21 @@
     align-self: center; // 4.
 }
 
-.ToastBar__message {
+.ToastBar__container {
```

Comment:
if we are going to change this, we need to do this everywhere. In the HTML as well. And README of course.